bundler/spec/realworld/slow_perf_spec.rb

```diff
@@ -17,6 +17,6 @@
 
     duration = Time.now - start_time
 
-    expect(duration.to_f).to be < 12 # seconds
+    expect(duration.to_f).to be < 18 # seconds
```

Comment:
This test just strikes me as being too unreliable in general. Maybe `skip if ENV['CI']`?

Reply:
Yes, indeed, but if it's unreliable in CI I don't see why it wouldn't be locally too. This spec was originally added to cover a performance issue where this Gemfile would take more than 10 minutes to resolve, so I think having something relaxed still covers us from a real regression here where the test would most likely fail consistently.

Reply:
I will merge this for now, but we can keep discussing this.